### ISSUE TITLE
AndroidTreeView with not showing padding for first level of hierarchy. Works for second level

### ISSUE THREAD

**Issue by ndubbaka** [bmelnychuk/AndroidTreeView#174]
I am using AndroidTreeView to show a hierarchy of items. First level of treeview is not being displayed with padding. the tree view like hierarchy is showing up from second level to all other levels. Bug explained in this video at https://www.youtube.com/watch?v=_9XjgVCqIOg&t=146

If I remove the arrow icon, it shows good as expected with the correct padding for all the levels. I need arrow icon to indicate that the user can expand & collapse. https://github.com/onedrupal/One-Drupal-Android/blob/master/app/src/main/res/layout/layout_node.xml

```
<com.github.johnkil.print.PrintView
        android:layout_alignParentLeft="true"
        android:id="@+id/arrow_icon"
```
https://github.com/onedrupal/One-Drupal-Android/blob/master/app/src/main/java/treeutil/ArrowExpandSelectableHeaderHolder.java#L55

```
arrowView = (PrintView) view.findViewById(R.id.arrow_icon);
        arrowView.setPadding(20,10,10,10);
        if (node.isLeaf()) {
            arrowView.setVisibility(View.INVISIBLE);
        }
```

**Comment by MonikaJethani** [comment]
@ndubbaka Is this still open? Can I work on this?

**Comment by ndubbaka** [comment]
@MonikaJethani Yes. The bug still exists in my code. I don't know what's wrong there and how to fix it.
